dashboard/test/factories.rb

```diff
@@ -393,6 +393,7 @@
         evaluator.section.try(:user) ||
         build(:teacher)
       follower.section = evaluator.section || build(:section, user: follower.user)
+      follower.user_id = follower.section.user_id
```

Comment:
Thus, without this, the attribute `follower.user_id` does not get set, causing lots of test failures as the validation `user_id == section.user.id` is not met.